packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -199,8 +200,22 @@ export const FeedContainer = ({
           aria-live={subject === ToastSubject.Feed ? 'assertive' : 'off'}
           data-testid="posts-feed"
         >
-          {router.query?.welcome === 'true' && (
-            <FeedReadyMessage className="mb-10" />
+          {showFeedReadyMessage && (
+            <FeedReadyMessage
+              className={{
+                main: classNames(
+                  shouldUseFeedLayoutV1
+                    ? 'mt-8 mb-8 w-full laptop:gap-4 [@media(width<=680px)]:px-6'
+                    : 'mb-10 max-w-xl laptop:gap-6',
+                ),
+                textContainer: classNames(
```

Comment:
I don't really mind, but think you can omit the use of `classNames` here.

Reply:
Hmm, I definitely can, will make it a bit shorter...